aksel.nav.no/website/app/dev/_ui/HoverCard.tsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import { ReactNode, useRef, useState } from "react";
+import { Button } from "@navikt/ds-react";
+import { Popover, PopoverContent } from "@navikt/ds-react/Popover";
+
+export const HoverCard = ({
+  popoverContent,
+  children,
+}: {
+  popoverContent: ReactNode;
+  children: ReactNode;
+}) => {
+  const buttonRef = useRef<HTMLButtonElement>(null);
+  const [openState, setOpenState] = useState(false);
+
+  return (
+    <>
+      <Button
+        ref={buttonRef}
+        onClick={() => setOpenState(!openState)}
+        aria-expanded={openState}
+        variant="tertiary-neutral"
+      >
+        {children}
+      </Button>
```

Comment:
Regular button does not work that great here visually, should sync with design. Supporting hovercard/popover isn't the most important for this task, so can skip to prioritize rest of implementation.

Reply:
Yeah, did we have an existing "hover" js implementation (dismissableLayer maybe)? I remember you said someone somewhere had a neat and simple impl using a timeout (that starts when you hover outside the element or its children),